### DISCUSSION TITLE
Kstar

### DISCUSSION THREAD

**Discussion by ahayashi-Gaus** [ai-republic/bms-to-inverter#84] [Q&A]
Hello,
We have a KSTAR BluE Series battery pack and inverter. Would be great to know if there is any way to decode the can messages between the inverter and the battery pack. I have a look for some of the CANIDs in other modules but didn't hit any matches.

**Comment by ai-republic** [comment]
I would need the CAN communication protocol specification to support it.

**Reply by ahayashi-Gaus** [reply]
And I would need to get that from the manufacturer correct?

**Reply by ai-republic** [reply]
In the best case yes 😄 
Or sometimes you can find it on the internet somewhere
